addon/components/au-modal.hbs

```diff
@@ -44,10 +38,11 @@
           {{/if}}
         </h1>
         <button
-          class="au-c-modal__close"
+          class="au-c-modal__close {{unless this.isClosable 'is-disabled'}}"
```

Comment:
UX question: is disabling the button better than not showing it at all? Does it depend on the use-case?

Also, while testing it was hard to see the difference between the disabled and non-disabled states (other than the cursor change).

Reply:
> UX question: is disabling the button better than not showing it at all? Does it depend on the use-case?

I think making an interaction component invisible when it or its parent is disabled will be confusing for the user.

> Also, while testing it was hard to see the difference between the disabled and non-disabled states (other than the cursor change).

Yup, I noticed that too. I think I'll give it a go today with a lighter `disabled` color value.